<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string/>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QComboBox" name="stateFilterBox">
         <property name="placeholderText">
          <string>State</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="dateFilterBox">
         <property name="placeholderText">
          <string>Date</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="nameFilterBox">
         <property name="placeholderText">
          <string>A-z</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="searchBar">
         <property name="text">
          <string/>
         </property>
         <property name="placeholderText">
          <string>Search...</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="searchButton">
         <property name="text">
          <string>Search</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="clearButton">
         <property name="text">
          <string>Clear</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QTableView" name="tasksTableView">
      <property name="contextMenuPolicy">
       <enum>Qt::CustomContextMenu</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
